lib/Ast.ml

```diff
@@ -34,58 +34,21 @@ let ( init
 
 (** Predicates recognizing special symbol identifiers. *)
 
-let is_prefix_id i =
-  match i with
-  | "!=" -> false
-  | _ -> ( match i.[0] with '!' | '?' | '~' -> true | _ -> false )
+module Char = struct
+  include Char
```

Comment:
I'm not a fan of "improving" modules like this, especially if the added functions wouldn't fit in the original module.
What do you think of naming it `Char_id`, like you did with `String_id` ?
I think the same for `Longident` and `Exp` (it would be the only module without `_id`) below.

Reply:
I agree for `Char`; for `Longident` and `Exp` I aim to squeeze in the other related functions defined in this file (is_simple, etc.) and merge it with `Expression`, so they won't only contain id functions hopefully. That's the original plan but if it ends up being too messy I might do what you suggest